src/h5web/toolbar/MatrixToolbar.tsx

```diff
@@ -0,0 +1,39 @@
+import { FiDownload } from 'react-icons/fi';
+import { useMatrixConfig } from '../vis-packs/core/matrix/config';
+import DownloadBtn from './controls/DownloadBtn';
+import Toolbar from './Toolbar';
+
+function MatrixToolbar() {
+  const currentSlice = useMatrixConfig((state) => state.currentSlice);
```

Comment:
The toolbar needs access to the `ndarray` of the current slice. That's why I need a zustand state, which I set in `MappedMatrixVis`, via a `useEffect` (just like `dataDomain`).